docs/advisors/checks/mysql_version_eol_57.md

```diff
@@ -0,0 +1,17 @@
+# Percona Server and MySQL 5.7 End Of Life
+## Description
+Percona Server and MySQL server 5.7 are declared End Of Life versions. The deadline is on  Oct 21, 2023. 
```

Comment:
I would rephrase this paragraph to:

Percona Server for MySQL Server version 5.7 will reach End-of-life on October 21st, 2023.
Percona does not provide support or issue hotfixes for retired releases.

Reply:
We'll improve this later; as for now, WIP on final messages about 5.7 EOL and our offers on it.